### ISSUE TITLE
Creating a Twitter Bot in Go

### ISSUE THREAD

**Issue by jonaylor89** [danistefanovic/build-your-own-x#577]
### Main programming language
Go

### Tutorial title
Creating a Twitter Bot in Go

### Tutorial URL
https://blog.jonaylor.xyz/twitter-bot/

### Category
* [ ] 3D Renderer
* [ ] Augmented Reality
* [ ] BitTorrent Client
* [ ] Blockchain / Cryptocurrency
* [x] Bot
* [ ] Command-Line Tool
* [ ] Database
* [ ] Docker
* [ ] Emulator / Virtual Machine
* [ ] Front-end Framework / Library
* [ ] Game
* [ ] Git
* [ ] Network Stack
* [ ] Neural Network
* [ ] Operating System
* [ ] Physics Engine
* [ ] Programming Language
* [ ] Regex Engine
* [ ] Search Engine
* [ ] Shell
* [ ] Template Engine
* [ ] Visual Recognition System
* [ ] Voxel Engine
* [ ] Web Search Engine
* [ ] Web Server
* [ ] Uncategorized

**Comment by PedroCisnerosSantana** [comment]
The link gives a 404, guess the link is https://blog.jonaylor.xyz/creating-a-twitter-bot-in-go

**Comment by jonaylor89** [comment]
Yes, that looks like the correct article. I guess the route changed

**Comment by PedroCisnerosSantana** [comment]
Nice, then this seems valid to me
